### ISSUE TITLE
Move cards to different boards

### ISSUE THREAD

**Issue by CalmAussie** [hjohnson12/Kanban-Tasker#20]
Love the Kanban board.

Would you be able to add the ability to replicate existing cards into other boards or at least have an option to link existing cards to a separate board once you no longer have a need for the whole board?

**Comment by andremoraes** [comment]
good idea, perhaps copy and paste of some sort should net be hard (or dropdown menu)
